### DISCUSSION TITLE
怎么加一行联合导师的姓名、职称？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#164] [Q&A]
你好，
读博有两个导师，怎么在“指导教师姓名、职称”下面添加一行“联合教师姓名、职称”？
谢谢

**Comment by xkwxdyy** [comment]
你先局部自己改一下吧，找到 `whu-thesis.cls` 的 1145 行，然后添加：
```latex
\__whu_spread_with:nn { 联合教师姓名、职称 } { \hfill }
     & <content>  \\ [ 0.5 em ]
```

**Reply by earthmoon** [reply]
> ```
> \__whu_spread_with:nn 
> 
联合教师的姓名、职称取哪里填呢？

**Reply by xkwxdyy** [reply]
```latex
<content>
```

**Reply by earthmoon** [reply]
> ```
> \__whu_spread_with:nn { 联合教师姓名、职称 } { \hfill }
>      & <content>  \\ [ 0.5 em ]
> ```

\__whu_spread_with:nn { 联合教师姓名、职称 } { \hfill }
            & <张三,教授>  \\ [ 0.5 em ]

我测试这个好像没动静哎。

**Reply by xkwxdyy** [reply]
我近期在重构模版，你要么就等最新的模版吧

**Reply by earthmoon** [reply]
> 我近期在重构模版，你要么就等最新的模版吧
好的，谢谢！
